qualtran/bloqs/for_testing/costing_test.py

```diff
@@ -0,0 +1,32 @@
+#  Copyright 2023 Google LLC
```

Comment:
Update 2023 to 2024 everywhere.

Reply:
gotta update my pycharm setting that puts these in automatically :)